migrations/2_deploy_contracts.js

```diff
@@ -80,6 +82,11 @@ async function deployProtocol(deployer) {
         return console.log(err);
       }
     });
+    fs.writeFile(path.join(homedir, '.enigma', 'votingethcontract.txt'), VotingETH.address, 'utf8', function(err) {
```

Comment:
You don't need this section, because when you need to know this address from within this container, you have direct access to the JSON of the compiled contract, like in [here](https://github.com/enigmampc/enigma-contract/blob/91fff28fcc290a82eb880087a9faf173022858c8/enigma-js/test/integrationTests/template.02_deploy_voting.js#L51). This is only needed for contract addresses that need to be shared between containers, but this one is only used here for testing purposes.

Removed in cc86d8c